runtests.py

```diff
@@ -9,8 +9,8 @@
 def runtests(*test_args):
     # Setup settings
     if not settings.configured:
-        from dynamic_raw_id.tests.testapp import settings as TEST_SETTINGS
-        settings.configure(**TEST_SETTINGS.__dict__)
+        from dynamic_raw_id.tests.testapp.settings import TEST_SETTINGS
```

Comment:
if you're wondering why I did this change and the one inside `settings.py` it's that in Django3 it's enforcing checks that all your settings keys are in UPPERCASE. Unfortunately the previous method of passing in the module `__dict__` passes in all of the dunder methods (e.g., `__name__`, `__doc__`, `__package__`, etc). I'm not 100% happy with the solution I've come up with because it duplicates the settings technically so if you can think of a smoother way please advise